pkg/daemon/daemon.go

```diff
@@ -500,17 +505,23 @@ func (dn *Daemon) syncNode(key string) error {
 	node = node.DeepCopy()
 	if dn.node == nil {
 		dn.node = node
+		glog.Infof("New Init")
```

Comment:
I think we should probably remove this since it's not very clear what New Init is. For debugging purposes, if you'd like to have it around, we can also move it to a v(4) (debug) level log. WDYT?

Reply:
I will remove this one, I don't think it is super helpful out of context.